dev.sh

```diff
@@ -2,5 +2,4 @@
 
 # run compose
 docker-compose kill
-docker-compose -f docker-compose.yml build # --no-cache
-docker-compose -f docker-compose.yml up
+docker-compose up --no-color 2>&1 | sed 's/^[^ ]*  *| //'
```

Comment:
This removes the docker-compose container name prefixes from log so it's easier to copy log text.